include/openmc/distribution_energy.h

```diff
@@ -61,8 +61,9 @@ class LevelInelastic : public EnergyDistribution {
   double sample(double E, uint64_t* seed) const override;
 
 private:
-  double threshold_;  //!< Energy threshold in lab, (A + 1)/A * |Q|
-  double mass_ratio_; //!< (A/(A+1))^2
+  double a_;
+  double b_;
+  double c_;
```

Comment:
Need to have comments on these data members so someone looking at the class declaration knows what they are.